### DISCUSSION TITLE
Upgrade to RevealJS 4.6?

### DISCUSSION THREAD

**Discussion by jimmy-narang** [quarto-dev/quarto-cli#7421] [Feature Requests]
### Description

Hi!

I wanted to request an upgrade to revealjs 4.6 from the current* 4.3. (Current = Quarto 1.4.455). Between the two releases, Reveal JS added some basic and long needed features, such as the ability to jump to a specific slide.  

I've done some basic testing, and simply dropping in the 4.6 dist seems to work just fine / doesn't break slides compiled with Quarto.  Would be great to have this built-in.

**Comment by cderv** [comment]
Tracked in 
* https://github.com/quarto-dev/quarto-cli/issues/4795
